tests/testthat/test-mle-finder.R

```diff
@@ -11,3 +11,17 @@ test_that("linalg and optim least-sq coincide", {
     coef(via_linalg_out), coef(via_bfgs_out), abs_tol = 1e-2, rel_tol = 1e-2
   ))
 })
+
+
+test_that("newton and bfgs outputs coincide on logit model", {
```

Comment:
-4

Test is failing. It's because of the above comments relating to implementing the negative log likelihood, and setting your threshold too loose for convergence.